priv/repo/migrations/20220712094523_create_user_skills.exs

```diff
@@ -0,0 +1,29 @@
+defmodule Bokken.Repo.Migrations.CreateUserSkills do
+  use Ecto.Migration
+
+  def change do
+    create table(:user_skills, primary_key: false) do
```

Comment:
Don't mix `mentor_skills` table with `ninja_skills` table. Use separate tables because that way you get cleaner code everywhere.

Reply:
That means I should create two distinct data types correct (on ```lib/bokken/accounts``` ?